### ISSUE TITLE
adding channel when clicking on channel list

### ISSUE THREAD

**Issue by martin4eel** [xmltv-se/issues#57]
Just a small problem, nothing super serious. 

So when I am in the "Edit Channels" page and click on the right hand side of any channel it automatically tick the first channel (13th Street (DE)). If I click again it un-ticks the channel. 

Not a big issuse, but I just noticed that I had added this channel to my XMLTV-file without knowing. 

Thanks for a great service 💯

**Comment by jnylen** [comment]
Hi @martin4eel 
It's kinda built that way. That page is just a temporary page as we had issues with the way we did it before. 
I'm currently finishing up a beta version of a project that HB and XMLTV.se require called [Metagraph](https://github.com/pixelmonster-ou/metagraph) and when that is done we will relaunch a new design and fix some issues there. But until then I will only fix some major issues.

But thanks for telling me :) 

Closing this as well as this is for the XMLTV.se project rather than HB :)
